packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -30,6 +30,7 @@ function validateNextProps(props, nextProps) {
  * @propType {func} [onSearchStateChange] - Function to be called everytime a new search is done. Useful for [URL Routing](guide/Routing.html).
  * @propType {object} [searchState] - Object to inject some search state. Switches the InstantSearch component in controlled mode. Useful for [URL Routing](guide/Routing.html).
  * @propType {func} [createURL] - Function to call when creating links, useful for [URL Routing](guide/Routing.html).
+ * @propType {object} [resultsState] - Object or Array to inject some results to be used at first rendering. Useful for [Server Side Rendering](guide/Server-side_rendering.html). This object or array comes from the `findResults` function. 
```

Comment:
is it an array when multi index, and object when single index? Might want to elaborate a little on that here.